gcp/cluster.tf

```diff
@@ -0,0 +1,103 @@
+resource "random_uuid" "cluster" {}
+
+locals {
```

Comment:
Are the 4 local vars used anywhere?
If not I'd say use them or lose them :)

Reply:
Ah, I had to remove their usage because GCP was complaining about the resource names. Good catch.